types.go

```diff
@@ -177,10 +178,14 @@ type fwpmFilter0 struct {
 	NumFilterConditions uint32
 	FilterConditions    *fwpmFilterCondition0
 	Action              fwpmAction0
-	ProviderContextKey  windows.GUID
-	Reserved            *windows.GUID
-	FilterID            uint64
-	EffectiveWeight     fwpValue0
+
+	// Only one of RawContext/ProviderContextKey must be set.
```

Comment:
You say one of the two must be set, but then you don't set it anywhere?

Reply:
We don't actually need to set it anywhere right now, but if we don't include it in the struct the syscall fails because of misaligned structs.
I added this comment in reference to the `union` in https://docs.microsoft.com/en-us/windows/win32/api/fwpmtypes/ns-fwpmtypes-fwpm_filter0